verify-matching-service-test-tool/publish.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'maven-publish'
+
+def build_number = System.getProperty('IDA_MSA_VERSION')
+
+publishing {
+    publications {
+        mavenZip(MavenPublication) {
+            artifactId = "matching-service-test-tool"
+            version "$build_number"
+            groupId = "uk.gov.verify"
+
+            artifact file('verify-matching-service-test-tool' + "$build_number" + 'zip')
```

Comment:
Isn't there a `-` between `tool` and the build number?

Reply:
Good catch. I have updated it